docs/_static/css/dark.css

```diff
@@ -70,6 +71,12 @@ code.docutils.literal.notranslate {
     color: #ddd;
 }
 
+/* highlight color search text */
+
+.rst-content .highlighted {
+    background: #f0978b;
+    box-shadow: 0 0 0 2px #f0978b;
+}
```

Comment:
> In the dark mode CSS file, adjust the search highlight color to be soft red: #F0978B (yellow is currently not readable)

This is more readable now but I think it could still be improved. The contrast between the text and the background seems a bit low.

![contract-metadata-dark-mode](https://user-images.githubusercontent.com/137030/133288885-9957318a-a24c-424f-8394-391a320eeff3.png)